installers/metrics/ansible/roles/pgo-metrics-preflight/tasks/preflight-prometheus.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Check if inventory file variables are defined for Prometheus
+  fail: msg="Please specify the value of {{item}} in your inventory file"
+  tags: always
+  when: lookup('vars', item, default='') == ''
+  loop:
+    - db_port
+    - pgbadgerport
```

Comment:
I see why these are needed at this time. We should probably (soon) move toward scraping known ports rather than ignoring known ports (that is, `keep` rather than `drop` in prometheus config).

Reply:
Agreed.  This will take coordination with the pgmonitor team since they now own/control the [default scrape configs](https://github.com/CrunchyData/pgmonitor/blob/9697c0d41e2a353f2dda4c353b272e8be570a935/prometheus/crunchy-prometheus.yml.containers).